docs/project/contribute.md

```diff
@@ -64,7 +64,7 @@ All Contributors uses Docusaurus to maintain our documentation website. For more
 To get started locally:
 - `yarn install`
 - `yarn start`
-- go to http://localhost:3000 (Refresh the page for your changes to be reflected)
+- go to `http://localhost:3000` (Refresh the page for your changes to be reflected)
```

Comment:
prefer to change to markdown link

Reply:
@sinchang The problem with that is the fact that it will be treated as a live URL which would inevitably return noise on [brokenlinkcheck](https://www.brokenlinkcheck.com/broken-links.php#status) which is what I tried to avoid doing this.

Reply:
@sinchang Do you have a better idea regarding that?